react/StarIcon/StarIcon.less

```diff
@@ -9,3 +12,36 @@
   50% { transform: scale(1.1); }
   100% { transform: scale(1); }
 }
+
+.svg {
+  position: relative;
+  top: 0.05em;
```

Comment:
Does this position work for non-rating usages of the `StarIcon`?

Reply:
Yes, it does, check it out:
![Stars in playground](https://user-images.githubusercontent.com/5409602/38784744-050e2d94-415a-11e8-92b1-575adce6c620.png)

Reply:
It's been adjusted to sit better alongside text in general, optically sitting on the baseline, so that's the assumption. Might be worth trying it out on your app if you're concerned.

Reply:
I believe this is a breaking change, as consumers are likely to already have position adjustments that now need to be updated/removed. Also looking at the playground before and after, it's off grid after the icons and ratings sections.